### PR TITLE
PWN-4720 - transaction details bottomsheet - added

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4720

## Description of Work

PWN-4720 - transaction details bottomsheet - added

## Screenshots 

Screenshot_20220831_142012

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/common/feature_toggles/toggles/remote/NewBuyFeatureToggle.kt [hunk 1]
```diff
@@ -7,5 +7,5 @@ class NewBuyFeatureToggle(
 ) : BooleanFeatureToggle(valuesProvider) {
     override val featureKey: String = "buy_feature"
     override val featureDescription: String = "Enable new buy feature"
-    override val defaultValue: Boolean = false
+    override val defaultValue: Boolean = true
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/common/feature_toggles/toggles/remote/NewBuyFeatureToggle.kt:10]: revert?

File: app/src/main/java/org/p2p/wallet/common/ui/bottomsheet/BaseDoneBottomSheet.kt [hunk 2]
```diff
@@ -23,16 +24,25 @@ abstract class BaseDoneBottomSheet : BottomSheetDialogFragment() {
...
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        binding = DialogBaseDoneBinding.inflate(inflater, container, false)
-        return binding.root
+        baseDialogBinding = DialogBaseDoneBinding.inflate(inflater, container, false)
+        val innerView = onCreateInnerView(inflater, container, savedInstanceState)
+        baseDialogBinding.viewInner.addView(innerView)
+        return baseDialogBinding.root
     }
 
+    abstract fun onCreateInnerView(
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/common/ui/bottomsheet/BaseDoneBottomSheet.kt:37]: oh yes, the inheritance problem
i wanted to write about it, about the problem of resolving multiple bindings - BaseDoneBottomSheet's and its children
but i guess you figured it out

File: app/src/main/res/layout/dialog_base_recycler_part.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_base_recycler_part.xml:9]: recyclerView -> recyclerViewBasePart or idk
all this inheritance and naming making me nervous, but i cant suggest anything more useful

File: app/src/main/res/layout/dialog_buy_details_part.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/wrappedImageView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_buy_details_part.xml:10]: imageViewWrapped

File: app/src/main/res/layout/dialog_buy_details_part.xml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginVertical="42dp"
+        android:importantForAccessibility="no"
+        android:src="@drawable/buy_details_image" />
+
+    <org.p2p.wallet.common.ui.widget.OptionsTextView
+        android:id="@+id/priceView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/dialog_buy_details_part.xml:18]: textViewPrice/ optionsTextViewPrice

**Review by gslevinkov — APPROVED** [review]

**Review by 1sakov — APPROVED** [review]

**Pull request merged**